experimental/auth-logic/wrappers/provenance_wrapper.go

```diff
@@ -50,3 +57,19 @@ func (p provenanceWrapper) EmitStatement() (UnattributedStatement, error) {
 		Contents: fmt.Sprintf(`expected_hash("%s::Binary", sha256:%s).`, applicationName,
 			expectedHash)}, nil
 }
+
+// GetAppNameFromProvenance parses a provenance file and emits the name of the
+// application it is about. This is useful for generating principal names,
+// for example.
+func GetAppNameFromProvenance(provenanceFilePath string) (string, error) {
+	provenance, provenanceErr := slsa.ParseProvenanceFile(provenanceFilePath)
```

Comment:
Indentation is inconsistent in this function. Does not `go fmt` fix these?

Reply:
didn't run it before, did now